### PR TITLE
[Merged by Bors] - Update linux_dependencies for Gentoo.

### PR DESCRIPTION
**Pull request by p3rsik**
# Objective

- Add dependencies installation instructions for Gentoo for `linux_dependencies.md`.

## Solution

- Edit the `linux_dependencies.md`.

### CODE REVIEW

**Comment by alice-i-cecile** [comment]
This looks about right to me :) Thanks for the contribution!

File: docs/linux_dependencies.md [hunk 1]
```diff
@@ -103,3 +103,8 @@ Note that this template doesn't add Rust to the environment because there are ma
 ```bash
    sudo zypper install libudev-devel gcc-c++
 ```
+
+## Gentoo
```
Comment (alice-i-cecile) [inline docs/linux_dependencies.md:107]:
```suggestion
## Gentoo

```

**Review by alice-i-cecile — APPROVED** [review]

**Review by james7132 — APPROVED** [review]
Not a Gentoo user, but a quick glance through the wiki suggests that this would be correct, and the package requirements are in line with the other distros.

**Pull request closed without merging**